openfecli/commands/gather.py

```diff
@@ -344,6 +370,7 @@ def gather(rootdir, output, report, allow_partial):
         'dg': _write_dg_mle,
         'ddg': _write_ddg,
         'dg-raw': _write_dg_raw,
+        'raw': _write_raw,
```

Comment:
This is a really confusing option - we now have two very similar methods that match the exact same naming.

"raw" gives you raw dGs for all individual replicas
"dg-raw" gives you average dGs and stdevs across all replicas

My take here (and in #558) is that "dg-raw" isn't "raw", it's already processed. I'm not going to stand here and say "we need to change everything", but we need a better way to clarify this to users. Right now if I read the docstring I get two different options that start with "X reports the raw results,".